Normalize handling of moderation across all fora (#129)
larseggert/draft-ietf-modpod-group-processes: draft-ietf-modpod-group-processes.md

non-hostile environment in which all may participate on an equal footing,
regardless of their position or past contributions.

## Terminology

Below we use the term "administrator" to refer to the people who
are assigned by the IESG to manage a particular communication
channel, such as a mailing list or a chat group.  For example, working

> group chairs are administrators.

Reviewer: ```suggestion
group chairs are administrators of all the participation channels their WG
uses, which typically includes mailing lists and chat channels, but might
also include collaborative tools such as GitHub or GitLab. Another example
of administrators are the "owners" of non-WG IETF mailing lists.
```

Just to flesh this out a bit more.

and discussions in other systems that the IETF or WGs have chosen to
employ, such as GitHub repositories, Wikis, or issue trackers.

The moderators are authorized to moderate all

> fora, including the IETF discussion and last-call mailing lists

Author: Yeah we do.

all WG and non-WG mailing lists, area mailing lists, and chat systems in
coordination with the owners (if there are any) of those fora.
Participants
should contact both fora owners and moderators when they have concerns,

> and moderators should only act when the owners do not respond in a

Author: Me neither.  That's my problem.  See next commit.

> and moderators should only act when the owners do not respond in a

Author: group manager?

timely fashion.  In general, their first act should be to contact the
fora owners, and then only act if the owners remain unresponsive.

In the case of plenary fora, such as the ietf list, last-call list, and
other channels in which all or a large portion of the IETF community
engage, moderators shall act as owners, and they will be the primary
contacts for complaints.

Forum owners and moderators shall act in a manner

> consistent with guidelines approved by the IESG.  In cases of

Author: I've moved this down into the appeals discussion and changed the heading name.

disagreement between the chairs and moderation team
over a moderation decision, the matter should be taken up
with the responsible area director for resolution, or the IETF chair
if a responsible area director cannot be determined or is not assigned.

It is not expected that the moderators will be monitoring every
IETF channel, but rather that participants may and group management
should flag concerns about disruptive behavior to the moderators and
owners of those channels. However,
the moderators should actively monitor a small set of key
participation channels, including, for example, the IETF discussion
and last-call mailing lists or the IETF plenary chat channel. The
moderators should decide which channels are in this set based on
their own judgment and community suggestions. (Note that some
participation channels, such as the examples given in the previous
sentence, have no chairs or other community leadership, so the
moderators must handle those.)

> 

Author: @sayrer I think I agree.

> 

Reviewer 2: I think we need to indicate that dealing with WG lists and the general channels like ietf@ are the priority job and the other channels are done on a best-effort basis. Otherwise we are giving the moderators an unbounded work load. That would be a replacement for the parenthesis.

> 

Reviewer 3: @becarpenter Yes, I agree. I said something similar here:

https://mailarchive.ietf.org/arch/msg/mod-discuss/ISiJZYP06lUd1Tx4yvBaWv7je-Q/

"The moderator team can decide how to deal with non-WG lists on a case-by-case basis."

But this parenthetical is not making that point clear.

> 

Author: I do not want to signal that abuse would be tolerated anywhere.

and discussions in other systems that the IETF or WGs have chosen to
employ, such as GitHub repositories, Wikis, or issue trackers.

The moderators are authorized to moderate all
fora, including the IETF discussion and last-call mailing lists,

> all WG and non-WG mailing lists, area mailing lists, and chat systems in
> coordination with the administrators (if there are any) of those fora.

Reviewer: ```suggestion
all WG and non-WG mailing lists, area mailing lists, and chat channels
or other collaborative tools used to conduct public IETF work. They do so in
coordination with the administrators (if there are any) of those fora.
```

and made public, but need not be documented in the RFC series.

The intent of this memo is to provide the widest possible freedom of
action to administrators and moderators, with a few constraints.
Examples of actions that could be taken include:

- Automated rate limiting mechanisms;
- Review and approval of submissions/messages;
- A private or public admonishment;
- Temporary or permanent bans in one or more fora.

We stress that these are only examples, and not in any way

> prescriptive. administrators and moderators are free to decide on
> these or other actions.

Reviewer: ```suggestion
prescriptive. Administrators and moderators are free to decide on
these or other actions.
```

So how do we get moderation consistency? I assume this RFC will tasks the moderator team to establish and maintain a set of moderation actions together with some guidance on how to apply them, and administrators are expected to follow that guidance? Because otherwise we're back to the status quo, where every forum does what it wants.

Content removal or redaction from IETF archives are not moderation
actions, and are therefore beyond the scope of this memo.

## Consistency and Conflict Resolution {#appeals}

Forum administrators and moderators shall act in a manner
consistent with guidelines approved by the IESG.  In cases of
disagreement between the administrators and the moderation team
over a moderation decision, the matter should be taken up
with the responsible area director for resolution, or the IETF chair
if a responsible area director cannot be determined or is not assigned.

> Any moderation decision can be appealed to the IESG by anyone,

Reviewer: I think it was added because someone was asking why the IESG is the start of the appeals chain.

> Any moderation decision can be appealed to the IESG by anyone,

Author: But I will revert this text, and then bring this up as a separate issue.

authored by Eliot Lear, Robert Wilton, Bron Gondwana and John R. Levine.
Robert Sayre authored
[draft-sayre-modpod-excellent](<https://datatracker.ietf.org/doc/draft-sayre-modpod-excellent/>),
which also originated ideas reflected in this work.  Pete Resnick provided the

> basis for how the moderators interact with list / forum leadership.

Reviewer: Also add Pete to the list below?

> basis for how the moderators interact with list / forum leadership.

Author: Sorry, which list?  I already added him to acknowledgments.
